string-manipulation-2/src/last-chars.js

```diff
@@ -0,0 +1,22 @@
+/* exported lastChars */
+function lastChars(length, string) {
+  var output = '';
+  for (var i = 0; i < string.length; i++) {
+    if (i < (string.length - length)) {
+      continue;
```

Comment:
You should make the `<` a `>` so that you can remove this and only use your `else` block code. In general, `continue` is to be avoided, it is usually an indicator that you have your logic backwards somehow.